tests/geometry/functional/common.py

```diff
@@ -6,12 +6,28 @@
 import torch
 
 
-def check_lie_group_function(module, op_name: str, atol: float, *args):
+def check_lie_group_function(module, op_name: str, atol: float, args, funcs=None):
```

Comment:
What is `args` now? It seems that these are the input tensors? If so, better rename so it's easier to figure out what they are. Also missing type information for this and a few other inputs to this function. 

Can you also add some description of why we need to pass `funcs`?

Reply:
+1